server/jira_test_util.go

```diff
@@ -4,25 +4,19 @@
 package main
 
 import (
-	"encoding/json"
+	"io"
 	"os"
 	"path/filepath"
 )
 
-func getJiraTestData(filename string) *JiraWebhook {
+func getJiraTestData(filename string) io.Reader {
```

Comment:
@levb I agree, fixing with your suggestion

Reply:
The reason for returning `io.Reader` here is because `ParseWebhook()` takes an `io.reader` for its one argument. By returning a `[]byte` here, we would be effectively reconverting the test data back into a `reader` using `bytes.NewReader(data)` before we call `ParseWebhook` in the test.

That being said, I'm fine with returning a `[]byte` here :+1: